scripts/download_resources.py

```diff
@@ -1,3 +1,15 @@
+# coding: utf-8
+r"""
+Inputs
+-------
+target_paths : list
+    List of table names from OEP to be downloaded
```

Comment:
```suggestion
    List of PosixPaths of table names from OEP to be downloaded
```

Printing `target_paths` I got this: 
```
[PosixPath('results/_resources/ts_feedin.csv'), PosixPath('results/_resources/ts_load_electricity.csv'), PosixPath('results/_resources/ts_load_electricity_vehicles.csv'), PosixPath('results/_resources/scal_costs_efficiencies.csv'), PosixPath('results/_resources/ts_efficiency_heatpump_small.csv'), PosixPath('results/_resources/scal_load_heat.csv'), PosixPath('results/_resources/ts_load_heat.csv')]
```